content/blog/oauth2-for-hpe-greenlake-data-services-cloud-console.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Implementing OAuth 2 Flow for Data Services Cloud Console's Client
+  Application "
+date: 2021-10-14T12:46:19.488Z
+author: Ron Dharma
+authorimage: https://gravatar.com/avatar/8102f2adeef249065ccb9c43b8866d17?s=192
+thumbnailimage: /img/dscc-icon-transparent.png
+tags:
+  - data-services-cloud-console
+---
+## HPE GreenLake API Security
+
+At the introduction of Data Services Cloud Console public REST API in this [blog](https://deploy-preview-707--hpe-dev-portal.netlify.app/blog/api-console-for-data-services-cloud-console/) **(Using HPE GreenLake Console's API Gateway for Data Services Cloud Console)**, I introduce the concept and the process for the user to obtain the client credential authorization. This concept is known as OAuth 2 client-credential authorization workflow. This particular grant type allows the client application to authenticate using separate credentials (client id and client secret) that is authorized inside the API Gateway menu using the HPE GreenLake user account (Resource Owner)
+
+![Rehash the flow of the GreenLake access token acquisition](/img/greenlake-api-access-flow.png "Client-Credentials process")
```

Comment:
Hi Ron,
I do see "CONFIDENTIAL | 26" in the bottom right of the image. 
I would suggest to remove the "CONFIDENTIAL" and Page Number in the slide and recapture the image for this flow diagram". 

In PPT, you can select the "INSERT" then "Header & Footer", and uncheck the box "Slide number" and "Footer". Then capture the screenshot to make it an image. Upload it in the CMS editor.